src/RIO/ByteString/Lazy.hs

```diff
@@ -1,141 +1,211 @@
 -- | Lazy @ByteString@. Import as:
 --
 -- > import qualified RIO.ByteString.Lazy as B.Lazy
+{-# LANGUAGE NoImplicitPrelude #-}
 module RIO.ByteString.Lazy
   (
   -- * The @ByteString@ type
     Data.ByteString.Lazy.ByteString
 
   -- * Introducing and eliminating 'ByteString's
-  , Data.ByteString.Lazy.empty
```

Comment:
I'm not a fan of including unrelated changes like renamings in a PR like this. To be honest, I preferred the original naming in the module. Would you be able to undo that renaming?